### PR TITLE
WFLY-9814 fix testsuite / build for JDK 9 & 10

### PR DESCRIPTION
**Pull request by ctomc**
https://issues.jboss.org/browse/WFLY-9814

Please make sure your PR meets the following requirements:
- [x] Pull Request title is properly formatted: `[WFLY-XYZ] Subject` or `WFLY-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue(s)
- [x] Pull Request contains description of the issue(s)
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted

### CODE REVIEW

File: legacy/configadmin/src/test/resources/logging.properties [hunk 1]
```diff
@@ -24,7 +24,7 @@
 #loggers=org.jboss.whatever,org.jboss.foo
 
 # Root logger level
-logger.level=DEBUG
+logger.level=OFF
```
Comment (stuartwdouglas) [inline legacy/configadmin/src/test/resources/logging.properties]: Should this really be off? Wouldn't it be better to have it on ERROR to help diagnose any problems

File: jpa/hibernate4_3/src/test/java/org/jboss/as/jpa/hibernate4/scan/ScannerTest.java [hunk 2]
```diff
@@ -159,14 +147,6 @@ protected void addPackageToClasspath(File... files) throws MalformedURLException
         Thread.currentThread().setContextClassLoader(classLoader);
     }
 
-    protected void addPackageToClasspath(URL... urls) throws MalformedURLException {
```
Comment (stuartwdouglas) [inline jpa/hibernate4_3/src/test/java/org/jboss/as/jpa/hibernate4/scan/ScannerTest.java]: Why has so much hibernate code and tests been removed? There does not seem to be any link to a JIRA explaining the reasoning.

File: testsuite/integration/clustering/src/test/resources/logging.properties [hunk 3]
```diff
@@ -31,7 +31,7 @@ logger.sun.rmi.level=WARNING
 logger.level=INFO
 
 # Root logger handlers
-logger.handlers=FILE, CONSOLE
```
Comment (stuartwdouglas) [inline testsuite/integration/clustering/src/test/resources/logging.properties]: These logging changes should be in a seperate PR

**Review by stuartwdouglas — DISMISSED** [review]
This PR should be broken up into related changes (i.e. logging changes should be seperate to JDK9 work). Both of these should be linked against a JIRA that is still current, WFLY-3854 was closed against 11.0.0.Beta1.

File: legacy/configadmin/src/test/resources/logging.properties (hunk 1 above)
Comment (ctomc) [reply legacy/configadmin/src/test/resources/logging.properties]: yes, it should be off, if you need to diagnose it, you enable it right than, in 99.999% of the cases when the testsuite runs, which is on CI, noone will ever look at this.

File: jpa/hibernate4_3/src/test/java/org/jboss/as/jpa/hibernate4/scan/ScannerTest.java (hunk 2 above)
Comment (ctomc) [reply jpa/hibernate4_3/src/test/java/org/jboss/as/jpa/hibernate4/scan/ScannerTest.java]: unused code, no other reason.

**Comment by ctomc** [comment]
@stuartwdouglas moved the logging changes to https://github.com/wildfly/wildfly/pull/10869
updated all commits point to new jira issue

File: jpa/hibernate4_3/src/test/java/org/jboss/as/jpa/hibernate4/scan/ScannerTest.java [hunk 4]
```diff
@@ -398,35 +269,6 @@ protected File buildNestedEarDir(File includeFile) {
         return testPackage;
     }
 
-    @Test
-    public void testHttp() throws Exception {
```
Comment (kabir) [inline jpa/hibernate4_3/src/test/java/org/jboss/as/jpa/hibernate4/scan/ScannerTest.java]: Why is this test being removed?

**Comment by stuartwdouglas** [comment]
@scottmarlow can you review the hibernate changes?

**Pull request merged**